analysis/src/CompletionBackEnd.ml

```diff
@@ -1119,7 +1125,7 @@ and getCompletionsForContextPath ~debug ~full ~opens ~rawOpens ~pos ~env ~exact
               ~pos ~scope ~debug ~prefix ~env ~rawOpens ~full completionPath
             |> TypeUtils.filterPipeableFunctions ~env ~full ~synthetic
                  ~targetTypeId:mainTypeId
-            |> List.filter (fun (c : Completion.t) ->
+            (* |> List.filter (fun (c : Completion.t) ->
```

Comment:
@zth I'm not sure why but if we don't follow the type alias, this filtering stops working. Any thoughts?

Reply:
Is that the cause of the lost completions? So commenting this out fixes those?

Reply:
Yes, exactly